src/main/java/com/redhat/devtools/lsp4ij/LanguageServerWrapper.java

```diff
@@ -952,12 +952,13 @@ public boolean canOperate(@NotNull VirtualFile file) {
         return serverDefinition.isSingleton();
     }
 
-    private LanguageServerLifecycleManager getLanguageServerLifecycleManager() {
+    private @NotNull LanguageServerLifecycleManager getLanguageServerLifecycleManager() {
         Project project = initialProject;
         if (project.isDisposed()) {
             return NullLanguageServerLifecycleManager.INSTANCE;
         }
-        return LanguageServerLifecycleManager.getInstance(project);
+        var manager = LanguageServerLifecycleManager.getInstance(project);
+        return manager == null? NullLanguageServerLifecycleManager.INSTANCE : manager;
```

Comment:
When do you have this usecase? It means that we will loose some logs?

Reply:
Happened on closing IJ
```
2024-04-19 19:46:18,515 [ 707863] SEVERE - #c.i.o.u.ObjectTree - Cannot invoke "com.redhat.devtools.lsp4ij.lifecycle.LanguageServerLifecycleManager.onStatusChanged(com.redhat.devtools.lsp4ij.LanguageServerWrapper)" because the return value of "com.redhat.devtools.lsp4ij.LanguageServerWrapper.getLanguageServerLifecycleManager()" is null
java.lang.NullPointerException: Cannot invoke "com.redhat.devtools.lsp4ij.lifecycle.LanguageServerLifecycleManager.onStatusChanged(com.redhat.devtools.lsp4ij.LanguageServerWrapper)" because the return value of "com.redhat.devtools.lsp4ij.LanguageServerWrapper.getLanguageServerLifecycleManager()" is null
	at com.redhat.devtools.lsp4ij.LanguageServerWrapper.stop(LanguageServerWrapper.java:445)
	at com.redhat.devtools.lsp4ij.LanguageServerWrapper.stop(LanguageServerWrapper.java:436)
	at com.redhat.devtools.lsp4ij.LanguageServerWrapper.dispose(LanguageServerWrapper.java:417)
```

Reply:
Ok thanks for your clarification.